FCCee/FullSim/ALLEGRO/ALLEGRO_o1_v03/run_digi_reco.py

```diff
@@ -686,23 +725,36 @@ def setupSWClusters(inputCells,
 
     # Clustering parameters
     # - phi-theta window sizes
-    windT = 9
-    windP = 17
-    posT = 5
-    posP = 11
-    dupT = 7
-    dupP = 13
-    finT = 9
-    finP = 17
-    # to be tested: about -2% of energy but smaller cluster, less noise
-    # windT = 7
-    # windP = 9
-    # posT = 5
-    # posP = 7
-    # dupT = 7
-    # dupP = 9
-    # finT = 7
-    # finP = 9
+    if clusterType == "ReducedSize":
+        # to be tested: about -2% of energy but smaller cluster, less noise
+        windT = 7
+        windP = 9
+        posT = 5
+        posP = 7
+        dupT = 7
+        dupP = 9
+        finT = 7
+        finP = 9
+    elif clusterType == "SmallSize":
```

Comment:
`SmallSize` and `ReducedSize` are quite similar in their meaning. What about changing `SmallSize` to something like `MuonSize`?